syntax = "proto3";

package databaseapp;

// The Application service definition.
service ApplicationServer {
    // Get simple response
    rpc getAllCompanies (EmptyRequest) returns (stream CompanyResponse) {}
}

// Empty request message
message EmptyRequest {
}

// The response message
message CompanyResponse {
    int32 id = 1;
    string name = 2;
    string address = 3;
}
